test/integration/builder/additional.js

```diff
@@ -256,9 +256,9 @@ module.exports = function(knex) {
     });
 
 
-    it('.timeout() should throw TimeoutError', function(done) {
+    it('.timeout() should throw TimeoutError', function() {
       var dialect = knex.client.config.dialect;
-      if(dialect === 'sqlite3') { return done(); } //TODO -- No built-in support for sleeps
+      if(dialect === 'sqlite3') { return; } //TODO -- No built-in support for sleeps
```

Comment:
you can make sqlite sleep implicitly by using two separate transactions/connections, one locks a database table, and another attempts to access that table, which will cause it to wait for the lock to be released (5 seconds by default, if I recall correctly)